README.md

```diff
@@ -1,2 +1,97 @@
-# Hello---micro
-First project with microverse
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
```

Comment:
- [ ] Optional
I will put this requirement here but it has nothing to do with this part. Kindly add meaningful messages when making commits. An example is `git commit -m "Add HTML file and CSS file"`, that way anyone going through your work will have an idea of where a particular change was introduced to the code just by checking your commit message
Your commit says `edit`, which doesn't necessarily explain anything
![image](https://user-images.githubusercontent.com/81932478/170241065-b73aa089-89ed-42e3-8cad-8dc54dcb77d3.png)

Reply:
dear i have done exactly what you asked me to do 
thanks @Munsa1